.github/lib/README.md

```diff
@@ -1,12 +1,12 @@
 ## libpq, libcrypto-3, libssl-3
 
-These DLLs are required for some PostgreSQL related dependencies on Windows. these are being packaged in the the installer and will be extracted in the fastn's installation directory on Windows.
+These DLLs are required for certain PostgreSQL-related dependencies on Windows. They are being packaged in the installer and will be extracted in the fastn's installation directory on Windows.
```

Comment:
These lines are too long. When looking at a PR in Github, they should not wrap like this.